jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/TaskAssignmentSummaryMapper.java

```diff
@@ -51,62 +51,61 @@ public TaskAssignmentSummary apply(final TaskInstance task,
 
     protected Boolean isForwardAllowed(final TaskInstance task,
                                        final IdentityProvider identityProvider) {
-        if ("Completed".equals(task.getStatus())) {
-            return false;
+        if (TaskStatus.TASK_STATUS_READY.equals(task.getStatus())) {
```

Comment:
the implementation of isForwardAllowed and  isDelegationAllowed seem identical. If the criteria are supposed to be the same for both, we should implement them in a single method, so that they don't diverge in time.

Consider introducing private method that would these two call (to keep the distinction between forwarding and delegating a task)

Reply:
@jstastny-cz, this duplication was related with maintenance purposes as they are reflecting different operations checks, but regarding currently they are checking the same conditions I unified them in your proposed private method. Could you take a look? Thanks!